safe/definitions/exposure_classifications.py

```diff
@@ -673,6 +673,101 @@
     ]
 }
 
+badan_geologi_landcover_classes = {
+    'key': 'volcanic_ash_landcover_classes',
+    'name': tr('Volcanic Ash Landcover Classification'),
```

Comment:
@myarjunar and @ivanbusthomi Is-it only specific for volcanic ash ? I guess no, we can use this landuse classification with tsunami,... no ?